taccsite_cms/static/site_cms/css/src/_imports/trumps/s-breadcrumbs.css

```diff
@@ -0,0 +1,62 @@
+/*
+Breadcrumbs
+
+A list of links and to ancestor navigation (pages or sections that are a parent, grandparent, et cetera of the current page or section).
+
+Notice: A scope class is used to enforce appropriate semantic HTML.
+
+Markup:
+<!-- major navigation -->
+<nav class="s-breadcrumbs">
+  <ol>
+      <li><a href="/"><span>Home</span></a></li>
+      <li><a href="/"><span>Science</span></a></li>
+      <li><span>Publications</span></li>
+  </ol>
+</nav>
+<!-- minor navigation -->
+<ol class="s-breadcrumbs">
+    <li><a href="/"><span>Home</span></a></li>
+    <li><a href="/"><span>Science</span></a></li>
+    <li><span>Publications</span></li>
+</ol>
+
+Styleguide Trumps.Scopes.Breadcrumbs
+*/
+@import url("_imports/tools/x-truncate.css");
+
+
+
+
+
+/* Base i.e. Container */
+
+nav.s-breadcrumbs {
+  padding-top: 15px;
+
+  font-size: var(--global-font-size--x-small);
```

Comment:
__Note__: This variable will be functional after an upcoming PR for global typography.

Reply:
Typography PR merged. This font size is effectual, now.